frontend/src/components/UsersList.js

```diff
@@ -0,0 +1,63 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import MockAxios from "axios-mock-adapter";
+
+const UsersList = () => {
+  const [users, setUsers] = useState([]);
+
+  const mock = new MockAxios(axios);
+
+  mock.onGet("/users").reply(200, {
+    users: [
+      {
+        id: 1,
+        name: "John",
+        email: "j@trade.com",
+        approvedUser: true,
+        dateOfRegistration: 2020,
+      },
+      {
+        id: 2,
+        name: "Karl",
+        email: "k@trade.com",
+        approvedUser: false,
+        dateOfRegistration: 2021,
+      },
+    ],
+  });
+
+  useEffect(() => {
+    axios.get("/users").then((response) => {
```

Comment:
Az api hívásokat ki lehetne szervezni egy külön fájlba a következő módon:
https://github.com/axios/axios#creating-an-instance

A Mockok is lehetnének egy külön fjálba amit az előbbi fájl húzna be.